tests/test_reconcile.py

```diff
@@ -31,7 +31,7 @@ def test_merge(self):
         msdf2 = parse_sssom_table(data_dir / "basic2.tsv")
         merged_msdf = merge_msdf(msdf1, msdf2)
 
-        self.assertEqual(71, len(merged_msdf.df))
+        self.assertEqual(122, len(merged_msdf.df))
```

Comment:
Seems odd they doubled in size?

Reply:
Some of the `mapping_justification` values were changed in such a way that I think that's expected.